### ISSUE TITLE
embedded/gems/2.1.1/gems/vagrant-2.1.1/plugins/providers/virtualbox/synced_folder.rb:48:in `block in enable': undefined method `[]' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by daniel-petric** [hashicorp/vagrant#9933]
### Vagrant version
Vagrant 2.1.1
### Host operating system
Windows 10 Pro

### Guest operating system
Ubuntu 14.04.5 LTS (GNU/Linux 3.13.0-87-generic x86_64)
Ubuntu + Fedora Repository :  https://github.com/fcrepo4-exts/fcrepo4-vagrant.git 

### Vagrantfile

```ruby
# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
VAGRANTFILE_API_VERSION = "2"

Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
  # All Vagrant configuration is done here. The most common configuration
  # options are documented and commented below. For a complete reference,
  # please see the online documentation at vagrantup.com.
	
	config.vm.hostname = "fedora4"

	config.vm.box = "fcrepo/fcrepo4-base"

	config.vm.network :forwarded_port, guest: 8080, host: 8080 # Tomcat
	config.vm.network :forwarded_port, guest: 9080, host: 9080 # Fixity and Reindexing

  config.vm.provider "virtualbox" do |v|
  	v.memory = 2048
  end

  shared_dir = "/vagrant"

  config.vm.provision "shell", path: "./install_scripts/fedora4.sh", args: shared_dir
  config.vm.provision "shell", path: "./install_scripts/backup_restore.sh", args: shared_dir
  config.vm.provision "shell", path: "./install_scripts/fedora_camel_toolbox.sh", args: shared_dir

end
```

### Debug output

https://gist.github.com/daniel-petric/707daae48dadbfa993c723706c5d3b3d

### Expected behavior

"vagrant up" command succesfully executed

### Actual behavior

Errors / calls stack displayed
C:/HashiCorp/Vagrant/embedded/gems/2.1.1/gems/vagrant-2.1.1/plugins/providers/virtualbox/synced_folder.rb:48:in `block in enable': undefined method `[]' for nil:NilClass (NoMethodError)
...

### Steps to reproduce

1. traces for Fedora repository redirected to vagrant shared folder
2. vagrant up

**Comment by chrisroberts** [comment]
Hi there,

Would you please provide a gist with the debug output from the command: `vagrant up --debug`

Thanks!

**Comment by daniel-petric** [comment]
Hello,

I had that error only once. It may have been triggered by some startup logging from Fedora Repository (a Tomcat7 war). Meanwhile I have updated Virtual Box and recreated the virtual machine :-(

vagrant up --debug
https://gist.github.com/daniel-petric/6b8d01e83dcef495fdb911e2c288c9ef

Thanks,
Daniel

**Comment by chrisroberts** [comment]
As the error is not reproducible I'm going to close this. If you do encounter the error again and can capture a debug log, please feel free to open an new issue.

Cheers!

**Comment by sankarvishnu** [comment]
Hi all,

I am getting the same exception, while running from vagrant from windows.

default: /vagrant => D:/Softwares/aerospike-vm
C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/plugins/providers/virtualbox/synced_folder.rb:48:in `block in enable': undefined method `[]' for nil:NilClass (NoMethodError)
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/plugins/providers/virtualbox/synced_folder.rb:36:in `each'
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/plugins/providers/virtualbox/synced_folder.rb:36:in `enable'
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/lib/vagrant/action/builtin/synced_folders.rb:93:in `block in call'
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/lib/vagrant/action/builtin/synced_folders.rb:90:in `each'
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/lib/vagrant/action/builtin/synced_folders.rb:90:in `call'
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/lib/vagrant/action/warden.rb:50:in `call'
        from C:/HashiCorp/Vagrant/embedded/gems/2.2.6/gems/vagrant-2.2.6/lib/vagrant/action/builtin/before_trigger.rb:23:in `call'
